index.d.ts

```diff
@@ -19,9 +19,11 @@ declare module "browserstack-local" {
     [key: string]: string | boolean;
   }
 
+  type LocalOptions = Partial<Options>;
```

Comment:
Don't see any another use of `LocalOptions`.

Why it should be moved to declared type

Reply:
When other frameworks are integrating the options for passing as arguments, they need to have an interface to declare. This is why I have pulled out this.